### ISSUE TITLE
Triple quote string literal descriptions cause late validation failure

### ISSUE THREAD

**Issue by thomasmichaelwallace** [sid88in/serverless-appsync-plugin#317]
First- thanks for all the work on this ace project!

AppSync's support for the GraphQL SDL is a little out of date. Specially it doesn't support string literals (triple quote) as descriptions: https://github.com/aws/aws-appsync-community/issues/38

If you use this [now best practace](https://spec.graphql.org/June2018/#example-64e5a) approach with this plug-in the schema validates (because, well, it is valid!) but your deployment will fail with:

```
ServerlessError: An error occurred: GraphQlSchema - Schema Creation Status is FAILED with details: Failed to parse schema document - ensure it's a valid SDL-formatted document..
```

Luckily I was aware of this problem, so I didn't lose too much time. But I think it would be nice if this plugin either:

1. Automatically stripped triple quote string literals before deploying the stitched schema [preferred]
2. Failed validation if the schema includes triple quote string literals.

**Comment by thomasmichaelwallace** [comment]
As part of my own deployment I'm solving the problem by pre-processing (removing the offending descriptions).

I'm more than happy to fold this into a PR if you agree it's something that should be part of this plugin.

**Comment by thomasmichaelwallace** [comment]
Actually - I think you might already be doing this, and I might have another problem.

Let me report back.

**Comment by thomasmichaelwallace** [comment]
For reference, this is what I needed to do get my schema to pass appsync's internal definition:

```js
  const conformSchema =
    (schema) => schema
      .replace(/"""[^"]*"""\n/g, '') // appsync does not support the new style descriptions
      .replace(/#.*\n/g, '') // appysnc does not support old-style # comments in enums, so remove them all
      .replace(/ *& */g, ', ') // appsync does not support the standard '&', but the "unofficial" ',' join for interfaces:
```

**Comment by bboure** [comment]
Fixed in #340
